linked-lists/tests/linkedList.test.js

```diff
@@ -176,6 +176,58 @@ describe('LinkedList class deserialize method tests', () => {
   });
 });
 
+describe('LinkedList class insertBefore method tests', () => {
+  let linkedList = new LinkedList();
+  linkedList.append(1);
+  linkedList.append(2);
+  linkedList.insertBefore(2, 5);
+
+  test('should increase the length property by one', () => {
+    let actual = linkedList.length;
+    let expected = 3;
+    expect(actual).toBe(expected);
+  });
+
+  test('inserted node should have proper next value', () => {
+    console.log(linkedList);

```

Comment:
Don't forget to delete console.logs when finished with them. (Wouldn't want to leave them in production code.)